osdep/mac/app_bridge_objc.h

```diff
@@ -24,6 +24,7 @@
 
 #include "options/m_config.h"
 #include "player/core.h"
+#include "common/global.h"
```

Comment:
Would it work if we instead forward declare it? UnsafeMutablePointer should work if fwd declarations?

`struct mpv_global;`

Reply:
i believe forward declarations aren't imported to swift so we would end up with an OpaquePointer again.

Reply:
> also i am not sure what 'best practice' is in those cases. we can also just merge https://github.com/mpv-player/mpv/pull/15396 and close this one.

I will merge #15396 either way, because it is good fix. This PR is good too, but `OpaquePointer` seems to be ok, but little bit annoying.

Reply:
sry, i didn't mean to imply that #15396 shouldn't be merged, it definitely should, just that this PR can safely be closed.

closing this in favour of your PR #15396. i am going to revisit all the swift parts and the OpaquePointer usage, and check if we have other such cases. will change those in one go then.